docs/source/quickstart.rst

```diff
@@ -0,0 +1,64 @@
+Quickstart
+==========
+
+Requirement
+-----------
+
+This project comprises a set of commands to be run at a shell command prompt.
+Examples used here are for a bash shell in an Ubuntu GNU/Linux environment.
+
+- `Python 3.9 <https://www.python.org/downloads/release/python-390/>`_, see the full list of dependencies in `environment.yml <https://github.com/NRCan/geo-deep-learning/tree/develop/environment.yml>`_
+- `hydra <https://hydra.cc/docs/intro/>`_
+- `mlflow <https://mlflow.org/>`_
+- `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ (highly recommended)
+- nvidia GPU (highly recommended)
+
+.. note::
+   
+   The system can be used on your workstation or cluster.
+
+.. _installation:
+
+Installation
```

Comment:
It would be nice to add docker image creation instructions from our Dockerfile. Adding this doc may not be the purpose of this PR, already quite big, but we can add this afterwards.

Reply:
Can you make a PR on that I will adress it after the merge of this PR.